src/Main.java

```diff
@@ -1,6 +1,85 @@
 public class Main {
     public static void main(String[] args) {
 
-
+        System.out.println("Задание 1");
+        int deposit1 = 15000;
+        int month = 0;
+        int pocket = 0; // в кармане
+        while (pocket < 2_459_000) {
+            pocket += deposit1;
+            month++;
+            System.out.println("Месяц " + month + ", сумма накомлений равна " + pocket + " рублей.");
+        }
+        System.out.println("Задание 2");
+        int number2 = 0;
+        while (number2 < 10) {
+            number2++;
+            System.out.print(number2 + " ");
+        }
+        System.out.println(" ");
+        int number2_1 = 10;
```

Comment:
зачем создавать новую переменную, если можно просто переиспользовать уже созданную?
```suggestion
        number2 = 10;
```